### PR TITLE
Create Practicing English.txt

### PR DESCRIPTION
**Pull request by mevian**

### CODE REVIEW

File: Practicing English.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+High priority
+-------------
+Harry potter
+
```
Comment (nefedovalexander80) [inline Practicing English.txt]: Ben-Naim A. Entropy Demystified - The Second Law Reduced to Plain Common Sense

Comment (nefedovalexander80) [inline Practicing English.txt]: [Ben-Naim A. Entropy Demystified - The Second Law Reduced to Plain Common Sense.pdf](https://github.com/Sumytech/reading-list/files/2223158/Ben-Naim.A.Entropy.Demystified.-.The.Second.Law.Reduced.to.Plain.Common.Sense.pdf)

Comment (nefedovalexander80) [reply Practicing English.txt]: I'd like to read it first,

**Comment by nefedovalexander80** [comment]
why did you delete my offer without any comment? I need to read this book anyway.

**Comment by mevian** [comment]
I've added a new pull request with the book listed. This GitHub repo is public, copyright issues etc.

**Pull request merged**